api/commands/run.ts

```diff
@@ -70,12 +75,18 @@ export class Runner {
       targetStringifier,
       graph.warehouseState.tables?.map(tableMetadata => [tableMetadata.target, tableMetadata])
     );
-
+    this.flags = {
+      runnerNotificationPeriodMillis: Flags.number("runner-notification-period-millis", runnerNotificationPeriodMillis),
+      displayRunningStatus: Flags.boolean("display-running-status", displayRunningStatus)
+    };
     this.executedActionTargets = new StringifiedSet(
       targetStringifier,
-      this.runResult.actions
-        .filter(action => action.status !== dataform.ActionResult.ExecutionStatus.RUNNING)
-        .map(action => action.target)
+      !!this.flags.displayRunningStatus.get() ?
```

Comment:
better to express this inside the filter:

```
this.runResult.actions
          .filter(action => this.flags.displayRunningStatus.get() || action.status !== dataform.ActionResult.ExecutionStatus.RUNNING)
```

(I actually think you got the flag the wrong way around here!)